components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -4677,7 +4677,7 @@ private static boolean validateEndpointURI(String endpointConfig)
                 JSONParser parser = new JSONParser();
                 JSONObject jsonObject = (JSONObject) parser.parse(endpointConfig);
                 Object epType = jsonObject.get("endpoint_type");
-                if(epType == null || "".equals(epType.toString().trim()) ){
+                if(StringUtils.isEmpty(epType.toString().trim())){
```

Comment:
I think we might not able to use StringUtils.isEmpty in here. Because if you trim a string without checking null it will cause null pointer exception. As previous if statement checking null first it won't cause null pointer.  As I remember this functionality can achieve through StringUtils.isBlank.